### DISCUSSION TITLE
RadioLib Release 1.0.0 is exactly 5 years ago today !

### DISCUSSION THREAD

**Discussion by ropg** [jgromes/RadioLib#1107] [General]
# 🥳

Congratulations Jan and all contributors!

**Comment by Velocet** [comment]
Congratulations and a huge thanks to @jgromes @StevenCellist @HeadBoffin and all other contributors 🙏🏻

**Comment by jgromes** [comment]
Quite the journey since then - here's hoping for a couple more good years :)
